tests/python_tests/test_generate_api.py

```diff
@@ -742,8 +742,8 @@ def test_unicode_pybind_decoding_3():
 ]
 
 configs = [
-    dict(max_new_tokens=500),
-    # dict(num_beam_groups=3, num_beams=15, num_return_sequences=15, max_new_tokens=20, diversity_penalty=1.0)
+    # dict(max_new_tokens=500),
+    dict(num_beam_groups=3, num_beams=15, num_return_sequences=1, max_new_tokens=500, diversity_penalty=1.0)
```

Comment:
should we enable both configs? 
is 500 required for func tests? can we have lower value to speed up tests execution?

Reply:
I committed this by mistake when i was checking beam search. But at the moment beam search fails on several calls. At the end i will uncomment both of them.

I will decrease max_new_tokens for speed.